### PR TITLE
Minds14

### PR DESCRIPTION
**Pull request by bahducoup**
Temporary PR for CI test

- TODO
  - [x] set default train configuration file to `train.yaml`

### CODE REVIEW

File: egs2/minds14/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+|inference_asr_model_valid.acc.ave_5best/valid|47|4178|66.8|20.2|13.0|19.2|52.5|100.0|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|inference_asr_model_valid.acc.ave_5best/test|49|8690|73.2|18.0|8.8|12.9|39.7|98.0|
+|inference_asr_model_valid.acc.ave_5best/valid|47|8751|74.3|15.7|10.0|15.6|41.3|100.0|
+
+### TER
```
Comment (ftshijt) [inline egs2/minds14/asr1/README.md:34]: If no TER is involved, you can just remove the section in readme

File: egs2/minds14/asr1/run_new.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```
Comment (ftshijt) [inline egs2/minds14/asr1/run_new.sh:1]: Please keep one `run.sh`

File: egs2/minds14/asr1/conf/smaller_conformer_wav2vec.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
+specaug: specaug
```
Comment (ftshijt) [inline egs2/minds14/asr1/conf/smaller_conformer_wav2vec.yaml:1]: It is cool to have many configs, but it seems that many of the configs are very similar to each other, could you try to combine them into fewer settings (e.g., the one without SSL features, the one with SSL features)

File: egs2/minds14/asr1/conf/slurm-2.conf [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+option name=* --job-name $0
+option time=* --time $0
+option mem=* --mem-per-cpu $0
+option mem=0
+option num_threads=* --cpus-per-task $0
+option num_threads=1 --cpus-per-task 1
+option num_nodes=* --nodes $0
+default gpu=0
+option gpu=0 -p RM-shared
+option gpu=* -p GPU-shared --gres=gpu:v100-16:$0 # Recommend allocating more CPU than, or equal to the number of GPU
```
Comment (ftshijt) [inline egs2/minds14/asr1/conf/slurm-2.conf:12]: the slurm need to be exactly the same as the template, please copy that from the `egs2/TEMPLATE`

**Review by ftshijt — COMMENTED** [review]
Thanks for your contribution

File: egs2/minds14/asr1/run_new.sh (hunk 2 above)
Comment (bahducoup) [reply egs2/minds14/asr1/run_new.sh:1]: Thanks for the feedback!
We are still running some experiments at the moment which is why we had multiple run.sh files.
We'll make sure to leave only one once the experiments are done.

**Comment by sw005320** [comment]
Are you planning to upload a model to the Hugging Face Hub?
If so, please add a link to the model to README.md

**Comment by karthik19967829** [comment]
> Are you planning to upload a model to the Hugging Face Hub?
> If so, please add a link to the model to README.md

Sure we are currently trying to get the best model using SSL , once done we will upload the model

**Comment by sw005320** [comment]
Please fix the conflict.

**Comment by bahducoup** [comment]
@sw005320 @ftshijt 
We've reflected all of the feedback to the PR as well as resolved the conflict.

File: egs2/minds14/asr1/run.sh [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+train_set="train"
+valid_set="valid"
+test_sets="test valid"
+language="es-ES"
+
+asr_config=conf/train.yaml
+
+./asr.sh \
+    --ngpu 1 \
+    --stage 1 \
```
Comment (simpleoier) [inline egs2/minds14/asr1/run.sh:17]: This line is not necessary.

File: egs2/minds14/asr1/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+   * `pt-PT`
+   * `ru-RU`
+   * `zh-CN`
+---
+
+# How to run
+* In `run.sh`, edit the value of `language` to `{lang}-{LOCALE}` (ex. `es-ES`, `en-AU`)
+---
+
+# RESULTS
```
Comment (simpleoier) [inline egs2/minds14/asr1/README.md:35]: Do you have more results (SSLR etc.) and corresponding pretrained models?

Comment (karthik19967829) [reply egs2/minds14/asr1/README.md:35]: Screen Shot 2022-04-04 at 10 20 14 PM

This has all the runs and results . Due to compute limitations we could not tune to get good performance with SSL.

**Comment by karthik19967829** [comment]
@simpleoier please let us know if anything is left to be added

File: egs2/minds14/asr1/run.sh (hunk 5 above)
Comment (karthik19967829) [reply egs2/minds14/asr1/run.sh]: Done

File: egs2/TEMPLATE/asr1/db.sh [hunk 7]
```diff
@@ -43,6 +43,7 @@ LIBRILIGHT_LIMITED=
 FSC=
 SLURP=
 VOXCELEB=
+MINDS14_DIR=${PWD}/MInDS-14
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/db.sh:46]: I don't think this PWD would be functioning

Comment (bahducoup) [reply egs2/TEMPLATE/asr1/db.sh]: @ftshijt Hi. I just fixed the path and confirmed that it functions as intended.

**Comment by bahducoup** [comment]
@simpleoier 
Hi, I just pushed a commit resolving the conflict in the README file.

**Pull request closed without merging**